components/Nav.jsx

```diff
@@ -0,0 +1,16 @@
+import { useContext } from "react";
+import { UserContext } from "../contexts/UserContext";
+import { Link } from "react-router-dom";
+
+const Nav = () => {
+    const { user } = useContext(UserContext);
+    
+    return (
+      <nav className="nav">
+        <Link to="/home"> Homepage </Link>
```

Comment:
the colour of this link does not have a high enough contrast on your background colour